legacy/firmware/config.c

```diff
@@ -575,13 +587,13 @@ const uint8_t *config_getSeed(void) {
       }
     }
     char oldTiny = usbTiny(1);
-    mnemonic_to_seed(mnemonic, passphrase, sessionSeed,
+    mnemonic_to_seed(mnemonic, passphrase, activeSessionCache->seed,
```

Comment:
Can we be absolutely sure that `activeSessionCache` is not` NULL`?

Reply:
Turns out we can't, it is the reason the segfault occurred that I've mentioned. If the Host sends Initialize before any call this should not happen, but we of course can't depend on that.